Jenkinsfile

```diff
@@ -1,26 +1,39 @@
 pipeline {
     agent { label 'mobile-builder-ios-pull-request'  }
-    environment {
-       PRODUCT = 'couchbase-lite-ios-ee'
-   }
     stages {
         stage('Checkout'){
             steps {
                 sh """
-                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git
-                    pushd ${env.PRODUCT}
+                    #!/bin/bash
+                    set -e
+                    shopt -s extglob dotglob
+
+		    # move PR related repo to tmp folder
+		    mkdir tmp
+                    mv !(tmp) tmp
+                    git clone https://github.com/couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET
+
+		    # restructure folders
+		    mv couchbase-lite-ios-ee/* .
+		    mv tmp/* couchbase-lite-ios
+		    
+		    # submodule update inside lite-ios
```

Comment:
I don't really know what's going on, here, but it seems odd that you check out the root directory without submodules and then go into one of the submodules and init it recursively.  Maybe a little comment that says what's going on, or something?

Reply:
I think @bmeike just pointed out the issue. After cloning the couchbase-lite-ios-ee, the submodule needs to be init and update. The following can be done when cloning couchbase-lite-ios-ee.

```git clone https://github.com/couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET -- recursive```